serve.go

```diff
@@ -203,11 +203,13 @@ func serve(args []string) error {
 
 	pushHandlers := nanopush.MakeHTTPHandlers(ctx, pushEndpoints, checkinOpts...)
 	scepHandler := scep.ServiceHandler(ctx, sm.scepService, httpLogger)
-	enrollHandler := enroll.ServiceHandler(ctx, sm.enrollService, httpLogger)
+	enrollHandlers := enroll.MakeHTTPHandlers(ctx, enroll.MakeServerEndpoints(sm.enrollService, sm.scepDepot), httptransport.ServerErrorLogger(httpLogger))
 	r := mux.NewRouter()
 	r.Handle("/mdm/checkin", mdmAuthSignMessageMiddleware(sm.scepDepot, checkinHandlers.CheckinHandler)).Methods("PUT")
 	r.Handle("/mdm/connect", mdmAuthSignMessageMiddleware(sm.scepDepot, connectHandlers.ConnectHandler)).Methods("PUT")
-	r.Handle("/mdm/enroll", enrollHandler).Methods("GET", "POST")
+	r.Handle("/mdm/enroll", enrollHandlers.EnrollHandler).Methods("GET", "POST")
+	r.Handle("/ota/enroll", enrollHandlers.OTAEnrollHandler)
+	r.Handle("/ota/phase23", enrollHandlers.OTAPhase2Phase3Handler).Methods("POST")
```

Comment:
Well, it neither starts, completes, or finalizes anything on its own. "StartAndFinalize" maybe? Its odd — the same URL is used for multiple steps (phases) in the OTA process. [This diagram](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/iPhoneOTAConfiguration/OTASecurity/OTASecurity.html#//apple_ref/doc/uid/TP40009505-CH3-SW1) is helpful but just keep in mind that phase 2 and 3 are both at the same URL.

Perhaps "enrollconfig" (Phase 2's name is "Device Enrollment" and Phase 3's name is "Device Configuration")?